edgemesh/pkg/server/tcp.go

```diff
@@ -1,26 +1,26 @@
 package server
 
 import (
-	"io"
-	"io/ioutil"
-	"net"
-	"net/http"
-
 	"github.com/go-chassis/go-chassis/core/common"
 	"github.com/go-chassis/go-chassis/core/handler"
 	"github.com/go-chassis/go-chassis/core/invocation"
 	"github.com/kubeedge/beehive/pkg/common/config"
 	"github.com/kubeedge/beehive/pkg/common/log"
 	"github.com/kubeedge/kubeedge/edgemesh/pkg/resolver"
+	"io"
+	"io/ioutil"
+	"net"
+	"net/http"
 )
 
 func StartTCP() {
-	server := config.GetString("server", "0.0.0.0")
-	port := config.GetString("port", "8080")
+	server, _ := getIP()
```

Comment:
Check err returned by getIP please, if err is not nil, refer to the startDnsServer function